mmdet/datasets/utils.py

```diff
@@ -98,7 +100,10 @@ def get_loading_pipeline(pipeline):
     """
     loading_pipeline_cfg = []
     for cfg in pipeline:
-        if cfg['type'].startswith('Load'):
+        obj_cls = PIPELINES.get(cfg['type'])
+        if obj_cls is None:
+            continue
+        if isinstance(obj_cls, (LoadImageFromFile, LoadAnnotations)):
```

Comment:
Cannot use instance, because it is not an instance relationship.  ->  `obj_cls in (LoadImageFromFile, LoadAnnotations)`, and above add 'TODO：use more elegant way to distinguish loading modules'

Reply:
like this：

    loading_pipeline_cfg = []
    for cfg in pipeline:
        obj_cls = PIPELINES.get(cfg['type'])
        # TODO：use more elegant way to distinguish loading modules
        if obj_cls is not None and obj_cls in (LoadImageFromFile, LoadAnnotations):
            loading_pipeline_cfg.append(cfg)